nixos/modules/misc/ids.nix

```diff
@@ -647,6 +648,7 @@ in
       paperless = 315;
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
+      hercules-ci-agent = 318;
```

Comment:
Do we need to allocate a uid and gid, or can we make use of `DynamicUser=yes` here?

Reply:
I don't think static allocation is strictly necessary. `DynamicUser` doesn't seem to be an option because it is not compatible with setuid workers. Although those aren't implemented yet, going back and forth will require the migration of state directories, twice.

Reply:
setuid workers seem to be a valid usecase for not using `DynamicUser` - but there's no need for static uid allocation.
Please only create the user using `users.users.hercuies-ci-agent` (without at static uid), and only if it's not the default.